### DISCUSSION TITLE
Copy the whole scrollback to editor for further usage

### DISCUSSION THREAD

**Discussion by wheatdog** [wezterm/wezterm#5294] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Is there a way to copy the whole scrollback to the default editor? I found this feature in zellij is quite useful. Wondering if we have the similar feature in wezterm.

https://zellij.dev/news/edit-scrollback-compact/#edit-your-scrollback-in-your-default-editor

**Comment by bew** [comment, marked as answer]
Checkout this example:
https://wezfurlong.org/wezterm/config/lua/wezterm/on.html#custom-events

**Comment by wheatdog** [comment]
Thanks for the pointer!
